setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name = "qark",
-    version = "1.2.19",
+    version = "1.2.19.1",
```

Comment:
GitHub suggests semantic versioning like `MAJOR.MINOR.PATCH`. Let's go with that guideline and have it be `1.2.20`.

Reply:
Semantic versioning also allows for the 4th number, which is for hotfixes 
that being said, I agree with 1.2.20 - will make changes now